test-nextjs/playwright.config.ts

```diff
@@ -77,5 +77,7 @@ export default defineConfig({
       : "npm run dev:frontend",
     url: "http://127.0.0.1:3000",
     reuseExistingServer: !process.env.CI,
+    stdout: "ignore", // Set to "pipe" if you're debugging a failing test.
```

Comment:
is it too loud? if this cost is just scrolling past logs seems useful to just have it?

Reply:
It adds about 100 lines of log output to a successful test run. But I guess others working on this might not even know to come in here and tweak this to debug a failing test - you're probably right that it's better to have on by default.